modules/_labs/core/react/package.json

```diff
@@ -39,6 +39,8 @@
   "dependencies": {
     "@emotion/core": "^10.0.21",
     "@workday/canvas-colors-web": "^0.17.13",
-    "@workday/canvas-kit-react-core": "^3.1.0"
+    "@workday/canvas-kit-react-core": "^3.1.0",
+    "chroma-js": "^1.4.0",
```

Comment:
Should we update `chroma-js` and types here? So we can also close #229 ?

Reply:
Are you wanting to re-write the color contrast function as part of this PR?